src/material/schematics/ng-add/theming/theming.ts

```diff
@@ -45,6 +47,23 @@ export function addThemeToAppStyles(options: Schema): (host: Tree) => Tree {
   };
 }
 
+/** Adds the global typography class to the body element. */
+export function addTypographyClass(options: Schema): (host: Tree) => Tree {
```

Comment:
Note: I can see this going into a separate file, but I kept it here because the typography system is tied to the theming.